Release notes automation (part III)

yangyansong-adbe opened the pull request.

dsoffiantini commented on .github/scripts/release_notes/android-release.js:
```diff
@@ -0,0 +1,195 @@
+
+    console.log(`request options: ${JSON.stringify(options)}`)
+
+    return new Promise((resolve) => {
```
This is probably easier to read if you use a package like `axios`

dsoffiantini commented on .github/scripts/release_notes/android-release.js:
```diff
@@ -0,0 +1,195 @@
+    console.log("fetchMavenArtifactInfo():")
+    console.log(array)
+    let releaseInfoArray = []
+    for (let i = 0; i < array.length; i++) {
```
This can be simplified with `.map`

praveek commented on .github/scripts/release_notes/update-release-notes.js:
```diff
@@ -0,0 +1,353 @@
+    // Read the contents of the markdown file.
+    let contentLines = fs.readFileSync(filePath, "utf8").toString().split("\n")
+    // Find the index of the release notes header.
+    let releaseNotesHeader = "# Release Notes"
```
# Release notes

praveek commented on .github/scripts/release_notes/update-release-notes.js:
```diff
@@ -0,0 +1,353 @@
+    return false
+}
+
+async function fetchNonAndoirdReleaseInfo(token, timestampInMilliseconds) {
```
typo: Android

praveek commented on src/pages/documentation/release-notes/index.md:
```diff
@@ -5,7 +5,7 @@ Keywords:
 ---
 
-# Release notes
+# Release Notes
```
This should be reverted.

yangyansong-adbe commented:
@dsoffiantini Thank you for the review comments! I added several TODOs in the JS code based on your comments. We discussed merging this PR for now and addressing your comments when I come back from my sabbatical.
